gulpfile.babel.js

```diff
@@ -10,12 +10,9 @@ import babel from 'gulp-babel';
 import concat from 'gulp-concat';
 import license from 'gulp-license';
 import rename from 'gulp-rename';
-import gulpSass from 'gulp-sass';
```

Comment:
Can you remove these from package.json too?

Reply:
Good point, I'll do that.